website/src/components/views/analyzeSingleVariant/SelectVariant.astro

```diff
@@ -0,0 +1,14 @@
+---
+import { PageHeadline } from '../../../styles/containers/PageHeadline';
+---
+
+<div class='mt-20 flex min-h-64 flex-col items-center'>
+    <PageHeadline>Analyze a single variant</PageHeadline>
+    <div class='max-w-xl'>
```

Comment:
I think this could use a little mx or px, so that the text has a little distance to the edge on small screens:
![grafik](https://github.com/user-attachments/assets/dc72032a-d7e7-44ac-a701-cb911198362a)